packages/server/utils/analytics/analytics.ts

```diff
@@ -107,8 +107,16 @@ class Analytics {
     })
   }
 
-  meetingStarted = (userId: string, meeting: Meeting, template?: MeetingTemplate) => {
-    this.track(userId, 'Meeting Started', createMeetingProperties(meeting, undefined, template))
+  meetingStarted = (
+    userId: string,
+    meeting: Meeting,
+    template?: MeetingTemplate,
+    isRecurring?: boolean
+  ) => {
+    this.track(userId, 'Meeting Started', {
+      isRecurring: isRecurring,
```

Comment:
I think `isRecurring` can be a common meeting properties (with default value `False` maybe):
1. I can see we'd also like to have that property for `Meeting Completed` events
2. Suppose in the future we can reuse this recurrence for other meeting types: "create a weekly retro meeting", won't that be cool?

Reply:
This is true, but this would effectively just be passing `isRecurring` into `createMeetingProperties`, just for `createMeetingProperties` to add it as a property, since we can't reliably determine whether a meeting is recurring just by the `meeting` object

1. I think `isScheduled` may be more representative, and also more future-proof, since it's less about the fact that the meeting's recurring and more that it was ended automatically. WDYT?
2. I hope so!

Reply:
> since we can't reliably determine whether a meeting is recurring just by the meeting object

Can you check if `meetingSeriesId` exists in the Meeting object?

Now I think about it more, I feel like instead of a boolean `isRecurring` or `isScheduled`, we should just inject `meetingSeriesId` as a common meeting property. It acts like a foreign key to the MeetingSeries table, just like `teamId`.

Reply:
Oops, you're right. Normally, we can't determine whether a meeting is actively recurring based on it's `meetingSeriesId`, but if we're starting a meeting that has a meetingSeriesId, then it's definitely being started as a result of recurrence.

Went ahead and added `meetingSeriesId` to the common meeting properties.